tools/image_processing/imagej2/imagej2_shadows/imagej2_shadows.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<tool id="imagej2_shadows" name="Add shadow effect" version="@WRAPPER_VERSION@.0">
+    <description>to images with ImageJ2</description>
```

Comment:
I guess we will install them into a category, called ImageJ, so this does not need to be part of the description. But I'm also not sure.